RecoLocalMuon/RPCRecHit/src/RPCCluster.cc

```diff
@@ -26,11 +26,11 @@ int RPCCluster::bx() const { return bunchx; }
 
 bool RPCCluster::hasTime() const { return nTime > 0; }
 float RPCCluster::time() const { return hasTime() ? sumTime/nTime : 0; }
-float RPCCluster::timeRMS() const { return hasTime() ? sqrt((sumTime2*nTime - sumTime*sumTime)/nTime) : -1; }
+float RPCCluster::timeRMS() const { return hasTime() ? sqrt(std::max(0.F, sumTime2*nTime - sumTime*sumTime)/nTime) : -1; }
```

Comment:
Is the problem appearing only for n=1?
It seems more appropriate to handle n=1 case explicitly.

Furthermore, applicable to yRMS as well as timeRMS: these are supposed to correspond to estimated rec hit y and t uncertainties. 
- I expect that the formula should be ```sqrt(   1/(n - 1) *  (  sumTime2 - sumTime*sumTime/n )  ) ``` from the unbiased estimator of the variance. There is 1/(n-1) term missing.
    - more hits should give a better cluster estimation, while the current formula e.g. for time mean = 0 gives a plain sqrt(sumTime2) which keeps growing.

Then, in addition, I do not see where the uncertainty of a single time or y measurement is added. Current version gives timeRMS = 0 in this case.